bot/cogs/bot.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, bot: AutoShardedBot):
                     formatted = f"```python\n{msg.content}\n```"
                     if len(formatted) <= 2000:
                         await msg.channel.send(formatted)
-                        await self.bot.delete_message(msg)
+                        await bot.delete_message(msg)
```

Comment:
It's `await msg.delete()`, in discord.py rewrite, all the actions upon an item are relative to the item itself.